styles/styles.css

```
@@ -717,11 +911,371 @@ body {
+}
+
+.product-card-button:active {
+  border: 2px solid #ff2f64;
+  background-color: #ffcbd8;
+}
+
+.product-card-button:active path {
+  fill: #2d3440;
+}
+
+.product-card-button:focus {
+  border: 2px solid #2d3440;
+  background-color: #ff2f64;
+}
+
+.footer-link-heart {
+  display: flex;
+  font-weight: 700;
+  gap: 8px;
+}
+
+.footer-link-heart::before {
+  content: '';
+  display: block;
+  flex-shrink: 0;
+  width: 16px;
+  height: 16px;
+  background-image: url('../images/heart.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.footer-link-product {
+  margin-left: 24px;
+}
+
+
+.other-taste-yellow {
+  position: relative;
+  display: flex;
+  width: 100px;
+  margin: auto 41px auto 37px;
+}
+
+.other-taste-yellow::before {
+  content: '';
+  position: absolute;
+  z-index: -1;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100px;
+  height: 100px;
+  margin: auto;
+  border-radius: 50%;
+  background: #ffcbd8;
+}
+
+.other-taste-pink {
+  position: relative;
+  display: flex;
+  width: 100px;
+}
+
+.other-taste-pink::before {
+  content: '';
+  position: absolute;
+  z-index: -1;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100px;
+  height: 100px;
+  margin: auto;
+  border-radius: 50%;
+  background: #ffcbd8;
+}
+
+.navigation-link:hover {
+  padding: 8px 16px;
+  border-radius: 30px;
+  background-color: rgb(252, 213, 223);
+}
+
+.navigation-link:active {
+  background: #fcfcfc;
+}
+
+.navigation-link:focus {
+  border-radius: 30px;
+  outline: 2px solid #2d3440;
+  background-color: rgb(252, 213, 223);
+}
+
+.slider-prev {
+  transform: rotate(180deg);
+  border: none;
+  border-radius: 50%;
+  background-color: #feafc3;
+}
+
+.slider-next {
+  transform: scaleY(-1);
+  border: none;
+  border-radius: 50%;
+  background-color: #feafc3;
+}
+
+.button-container {
+  margin-left: auto;
+  padding: 10px 0;
+}
+
+.button-container:first-child .page-catalog {
+  width: 10px;
+}
+
+.pacifier {
+  width: 142px;
+}
+
+.page-catalog {
+  font-family: Inter;
+  font-size: 16px;
+  border: none;
+  border-radius: 50%;
+  background-color: #feafc3;
+}
+
+.page-catalog:hover {
+  background-color: #fcd5df;
+}
+
+.page-catalog:active {
+  border: 1px solid #dbdbdb;
+  background-color: #fcd5df;
+}
+
+.page-catalog:focus {
+  border: 1px solid #2d3440;
+  background-color: #fec7d5;
+}
+
+.page-catalog:current {
+  background-color: #dbdbdb;
+}
+
+.field {
+  border: none;
+  border-radius: 4px;
+  outline: 1px solid #b9b9b9;
+  background: #fff;
+}
+
+.field:hover {
+  border: none;
+  border-radius: 4px;
+  outline: 1px solid #2d3440;
+}
+
+.field:focus {
+  border: none;
+  border-radius: 4px;
+  outline: 2px solid #2d3440;
+  background: #fff;
+}
+
+.field:disabled {
+  border: none;
+  border-radius: 4px;
+  outline: 1px solid #b9b9b9;
+  background: #e7e7e7;
+}
+
+.field:disabled:hover {
+  border: none;
+  border-radius: 4px;
+  outline: 1px solid #b9b9b9;
+  background: #e7e7e7;
+}
```

```suggestion
.field {
  border: none;
  border-radius: 4px;
  outline: 1px solid #b9b9b9;
  background: #fff;
}

.field:hover {
  outline-color:  #2d3440;
}

.field:focus {
  outline-color: #2d3440;
}

.field:disabled {
  outline-color: #b9b9b9;
  background: #e7e7e7;
}

.field:disabled:hover {
  outline-color: #b9b9b9;
  background: #e7e7e7;
}
```

Аналогично можно везде проверить так и поправить `border` -> `border-color` если только цвет меняется